src/components/Comment/Comment.tsx

```diff
@@ -531,69 +522,65 @@ export const Comment = ({
 								}}
 							/>
 							<div className={spaceBetween}>
-								<div className={shiftLeft}>
-									<Row>
-										{/* When commenting is closed, no reply link shows at all */}
-										{!isClosedForComments && (
-											<>
-												{/* If user is not logged in we link to the login page */}
-												{user ? (
-													<div className={svgOverrides} id="fgsghdiusfhfdsiu">
-														<ButtonLink
-															pillar={pillar}
-															onClick={() => setCommentBeingRepliedTo(comment)}
-															icon={<SvgIndent />}
-															iconSide="left"
-															linkName="reply to comment"
-														>
-															Reply
-														</ButtonLink>
-													</div>
-												) : (
-													<div
-														className={cx(
-															svgOverrides,
-															commentControlsLink(pillar),
-														)}
+								<Row>
+									{/* When commenting is closed, no reply link shows at all */}
+									{!isClosedForComments && (
+										<>
+											{/* If user is not logged in we link to the login page */}
+											{user ? (
+												<div className={svgReplyArrow}>
+													<ButtonLink
+														pillar={pillar}
+														onClick={() => setCommentBeingRepliedTo(comment)}
+														icon={<SvgIndent />}
+														iconSide="left"
+														linkName="reply-to-comment"
```

Comment:
is this change important? unlikely but it might break a report somewhere

Reply:
If you mean the link name, I think this can be reverted and was mainly something @liywjl implemented to make it consistent with the rest of the link names.